File: login_wave.java
package login1;
//import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
//import org.openqa.selenium.WebElement;
//import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
//import org.openqa.selenium.firefox.FirefoxDriver
//import org.openqa.selenium.Keys
public class login_wave {

	public static void main(String[] args)throws Exception{
		
		// open Chrome browser
		System.setProperty("webdriver.chrome.driver", "D:\\Drivers\\chromedriver_win32\\chromedriver.exe");

		WebDriver driver = new ChromeDriver();
      //  driver.manage().deleteAllCookies();
        driver.get("http://www.testwave.qabidder.net/");
        Thread.sleep(2000);
        driver.findElement(By.id("exampleInputEmail1")).sendKeys("smi16p+55@gmail.com");   
        driver.findElement(By.id("exampleInputPassword1")).sendKeys("test123");
         Thread.sleep(1000);      
        // click on the login button
        driver.findElement(By.xpath("//button[@type='submit']")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("//ul/li[2]/a/span")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/div[1]/div[1]/input")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/div[1]/div[1]/input")).sendKeys("suri");
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/div[1]/div[2]/input")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/div[1]/div[2]/input")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/div[1]/div[2]/input")).sendKeys("smi");
        Thread.sleep(1000);
        System.out.println("success");
       
        // click on view button
        driver.findElement(By.xpath("//tr[1]/td/div/div/div/div[6]/a")).click();
        Thread.sleep(1000);
        // click on add report
        driver.findElement(By.xpath("//span[contains(.,'Add Report')]")).click();
        Thread.sleep(1000); 
        // select radio button for organization
        driver.findElement(By.xpath("html/body/div[2]/section/div/div/div/form/div/div[2]/fieldset/div[1]/div/div/label")).click();
       // driver.findElement(By.("ng-binding")).click();
        Thread.sleep(1000);
        System.out.println("printing");
                //element.click();
        
         // click on next button
        driver.findElement(By.xpath("//div/div[2]/ul/li/a/span")).click();
        Thread.sleep(1000);
        // appearance page 
        // Click on First  name
        driver.findElement(By.xpath("//input[contains(@name,'first-name')]")).click();
        Thread.sleep(1000);
        driver.findElement(By.xpath("//input[contains(@name,'first-name')]")).sendKeys("priya");
        Thread.sleep(1000);
        driver.findElement(By.xpath("//input[contains(@name,'last-name')]")).click(); 
        Thread.sleep(1000);
        driver.findElement(By.xpath("//input[contains(@name,'last-name')]")).sendKeys("dutt"); 
        Thread.sleep(1000);
         driver.findElement(By.xpath("//input[contains(@name,'last-name')]")).click(); 
        Thread.sleep(1000);
        driver.findElement(By.xpath("//option[contains(.,'Suspect')]")).click();  
        Thread.sleep(1000);
        
         
        
        //Signinbutton.click();
        // close the browser
        driver.close();
        System.out.println("test script execute successfully.");
        // terminate the program
//        System.exit(0);      

               
		
		
		
	}

}

File: login_wave.py
from selenium import webdriver
from selenium.webdriver.support.ui import WebDriverWait
import logging
import time
import unittest
from datetime import datetime
class LoginTest(unittest.TestCase):
    logging.basicConfig(filename="C:\Users\Madhu\python\workspace_python\Wavefit\Test_suitePackage\TestSuitscreenshots\log.txt",
        level=logging.DEBUG)
    logging.debug('debug message')
    def __init__(self, *args, **kwargs):
        unittest.TestCase.__init__(self, *args, **kwargs)
        self.driver = webdriver.Chrome()
        # Navigate to login page
        self.driver.get("http://testwave.qabidder.net/")

    def setup(self):
        self.driver.maximize_window()

    def test_AddReport(self):
        driver = self.driver
        self.driver.maximize_window()
        validemail = "smi16p+55@gmail.com"
        password = "test123"
        driver.find_element_by_id("exampleInputEmail1").clear()
        driver.find_element_by_id("exampleInputEmail1").send_keys(validemail)
        driver.find_element_by_id("exampleInputPassword1").clear()
        driver.find_element_by_id("exampleInputPassword1").send_keys(password)
        time.sleep(2)
        driver.find_element_by_xpath("//button[@type='submit']").click()
        time.sleep(2)
        # click on logout button
        driver.find_element_by_xpath("//a[contains(text(),'Logout')]").click()
        time.sleep(2)

    def teardown(self):
        newtime = datetime.now().strftime("%Y%m%d%H%M%S")
        self.driver.save_screenshot('C:\Users\Madhu\python\workspace_python\Wavefit\Test_suitePackage\TestSuitscreenshots\screenshots' + newtime + '.png')
        self.driver.quit()

if __name__ == "__main__":
        unittest.main()
